rts/lra/service-base/src/main/java/io/narayana/lra/LRAConstants.java

```diff
@@ -42,7 +42,14 @@
     public static final String RECOVERY_PARAM = "recoveryCount";
     public static final String HTTP_METHOD_NAME = "method"; // the name of the HTTP method used to invoke participants
 
-    public static final long PARTICIPANT_TIMEOUT = 2; // number of seconds to wait for requests
+    /**
+     * System property name which defines the connection read timeout which is used
+     * by LRA Coordinator and its friends when connecting remote participants and other coordinators.
+     * The value is in seconds. The default value is {@value LRAConstants#PARTICIPANT_DEFAULT_TIMEOUT_S}.
+     */
+    public static final String PARTICIPANT_TIMEOUT_PARAM = "lra.participation.timeout";
+    private static final long PARTICIPANT_DEFAULT_TIMEOUT_S = 2; // number of seconds to wait for requests
```

Comment:
It seems odd to declare a private constant that is only used once and then only on the next line. The less code in an application the easier it is to understand and maintain.